payload/reggae/backend/ninja.d

```diff
@@ -84,7 +79,17 @@ NinjaEntry[] defaultRules(in Options options) @safe pure {
 
     NinjaEntry createNinjaEntry(in CommandType type, in Language language) @safe pure {
         string[] paramLines = ["command = " ~ Command.builtinTemplate(type, language, options)];
-        if(hasDepFile(type)) paramLines ~= ["deps = gcc", "depfile = $out.dep"];
+        if(hasDepFile(type)) {
+            version(Windows)
+                const isMSVC = language == Language.C || language == Language.Cplusplus;
+            else
+                enum isMSVC = false;
+
+            if (isMSVC)
+                paramLines ~= "deps = msvc";
```

Comment:
MSVC of course has its own format, to be used with `cl.exe  /showIncludes`.

Reply:
Of course it does.